learnCSharp/Commands.cs

```diff
@@ -122,26 +194,36 @@ public async static Task<Todo> UpdateStatus(ITodoRepository todoRepository, Stri
                 newTodo = await todoRepository.GetTodo(int.Parse(secondaryInput));
                 Console.WriteLine("ENTER STATUS(true or false):");
                 secondaryInput = Console.ReadLine();
-                newTodo.isDone = bool.Parse(secondaryInput);
+                newTodo.Completed = bool.Parse(secondaryInput);
             }
             return newTodo;
         }
 
 
-        public static async Task PrintSpecific(ITodoRepository todoRepository, String[] input)
+        public static async Task PrintSpecific(ITodoRepository todoRepository, String[] input, IUserRepository userRepository)
         {
+            var allUsers = await userRepository.GetAllUsers();
+            var userLookUp = allUsers.ToDictionary(user => user.ID, user => user.Name);
```

Comment:
Since we're only looking up one Todo here, there's no need to load all users, just the single user with the UserId in todo.